src/main/java/com/dubture/jenkins/digitalocean/DigitalOcean.java

```diff
@@ -3,6 +3,7 @@
  *
  * Copyright (c) 2015 Rory Hunter (rory.hunter@blackpepper.co.uk)
  *               2016 Maxim Biro <nurupo.contributions@gmail.com>
+ *               2017, 2021 Harald Sitter <sitter@kde.org>
```

Comment:
Can you undo this? The code is MIT licensed. If you are Harald Sitter, and don't want to have your code MIT licensed, then we won't be able to accept it and its not really open source.